src/widgets.tsx

```diff
@@ -534,7 +640,373 @@ export const Dropdown_Select = (props:{validationLabel?:string, setSelected:((va
    
 }
 
-export const ProfileImage = (props:{style?:ImageStyle}):JSX.Element => {
+export const Multi_Dropdown_Select = (props:{validationLabel?:string, setSelected:((val:string) => void), data: SelectListItem[], placeholder?:string, boxStyle?:ViewStyle, validationStyle?:TextStyle, defaultOption?:SelectListItem[] }):JSX.Element => {
+
+    const styles = StyleSheet.create({
+        dropdownText: {
+            color: COLORS.white,
+            textAlign: "center",
+            
+        },
+        dropdownSelected: {
+            color: COLORS.white,
+            textAlign: "center",
+            flex: 1 
+        },
+        dropdown: {
+            width: 300,
+            marginLeft: 3,
+            paddingVertical: 5,
+            paddingHorizontal: 15,
+        },
+        containerStyle: {
+            marginVertical: 5,
+        },
+        errorTextStyle: {
+            ...theme.accent,
+            color: COLORS.primary,
+            textAlign: "center",
+            marginBottom: 5,
+            ...props.validationStyle
+        },
+          
+    });
+    
+    return (
+        <View style={styles.containerStyle}>
+            <MultipleSelectList 
+                setSelected={(val: string) => props.setSelected(val)} 
+                data={props.data}
+                save="value"
+                boxStyles={props.boxStyle} 
+                dropdownTextStyles={styles.dropdownText}
+                inputStyles={styles.dropdownSelected}
+                placeholder={props.placeholder}
+                // TODO - defaultOption for SelectList is not available yet. PR - https://github.com/danish1658/react-native-dropdown-select-list/pull/102
+                //defaultOption={props.defaultOption} 
+             />
+            {props.validationLabel && <Text style={styles.errorTextStyle}>{props.validationLabel}</Text>}
+
+        </View>
+        
+      )
+   
+}
+
+export const FormInput = forwardRef<FormSubmit, FormInputProps>((props, ref):JSX.Element => {
+
+    // Determine if the field value is a string or a list by defining a type guard
+    // documentation: https://www.typescriptlang.org/docs/handbook/2/narrowing.html#the-in-operator-narrowing
+    const fieldValueIsString = (fieldType: InputType, value: string | string[]): value is string => {
+        return !isListType(fieldType);
+    }
+
+    const createFormValues = ():Record<string, string | string[]> => {
+        const formValues: Record<string, string | string> = {};
+        props.fields.forEach((field:InputField) => {
+            if (!fieldValueIsString(field.type, field.value || "")) 
+                if (field instanceof InputSelectionField || InputType.CIRCLE_ID_LIST || InputType.USER_ID_LIST) {
+                    //@ts-ignore
```

Comment:
With `props.defaultValues:any` shouldn't be needed anymore.

Reply:
done